db/migrate/20161117120104_create_organisations.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOrganisations < ActiveRecord::Migration[5.0]
```

Comment:
They will be useful at some point and are an integral part of this entity.. should we really remove this